index.js

```diff
@@ -1,3 +1,9 @@
-var wrapper = require('./wrapper.js');
+var fs = require('fs');
 
-module.exports = wrapper(require('./soljson.js'));
+try {
+  fs.lstatSync('./soljson.js');
+  var wrapper = require('./wrapper.js');
+  module.exports = wrapper(require('./soljson.js'));
+} catch (e) {
+  console.error('soljson.js not available.');
```

Comment:
I think we should not assume that `index.js` is used from a context that has a console available (the same applies to the wrapper). The whole package may be used through browserify.